frontend/.env.example

```
@@ -6,4 +6,7 @@ NEXT_PUBLIC_SITE_URL=https://telkom-alumni.vercel.app
 # GitHub API Configuration
 GITHUB_REPO_OWNER=fardhanrasya
 GITHUB_REPO_NAME=telkom-alumni
-GITHUB_TOKEN=your-github-personal-access-token
\ No newline at end of file
+GITHUB_TOKEN=your-github-personal-access-token
+
+# Revalidation Secret for Webhooks
+REVALIDATE_SECRET=your-super-secret-revalidation-key
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)